knowledge_storm/rm.py

```diff
@@ -333,10 +333,77 @@ def forward(self, query_or_queries: Union[str, List[str]], exclude_urls: List[st
         return collected_results
 
 
+class StanfordOvalArxivRM(dspy.Retrieve):
```

Comment:
Add a comment to say it's for internal use only (maybe say as "alpha"), not for public.